src/store/mutations.js

```diff
@@ -47,8 +47,17 @@ export default {
   setEdittingItemIndex(state, index) {
     state.edittingItemIndex = index;
   },
-  setCurrentSingleItem(state, item) {
-    state.currentSingleItem = item;
+  setCurrentItemObj(state, item) {
+    state.currentItemObj = item;
+  },
+  updateItemRelatedUnitsLocally(state, { field, value }) {
+    state.currentItemObj[field] = value;
+  },
+  resetRelatedUnitsLocally(state) {
+    if (state.currentItemObj) {
+      state.currentItemObj.relatedItems = null;
+      state.currentItemObj.relatedLists = null;
+    }
```

Comment:
inconsistent names "relatedUnits" and "ItemRelatedUnits"
I think "item" is redundant here